src/app/add-modules/add-module-modal/add-module-modal.component.scss

```diff
@@ -6,20 +6,20 @@
 }
 
 
-.add-module-dialog {
+.add-dialog {
   font-family: $font-family;
   border-radius: 4px 4px 0px 0px;
 
-  .module-input {
+  .add-dialog__content-input {
```

Comment:
&__content-input {

Reply:
![image](https://user-images.githubusercontent.com/108523765/179369148-7b5b9762-40b4-45a4-802c-f97cb241d358.png)

it is giving an error ana. I did not know how to use it

Reply:
don't put fullstop `.&__`. just like this `&__`